e2e/tests/dashboard-tests.spec.ts

```diff
@@ -1,35 +1,15 @@
 import { expect, test } from "#e2e/fixtures.ts";
 
-test.describe("Dashboard Module", () => {
-  test.beforeEach(async ({ page }) => {
-    await page.goto("/");
-  });
-
-  test("Dashboard widgets and actions", async ({ page }) => {
-    /*
-      Verify Quick Actions widget
-    */
-    await expect(page.getByText("Quick Actions")).toBeVisible();
-
-    // Verify "Record Attendance" button in Quick Actions
-    const recordAttendanceButton = page.getByRole("cell", {
-      name: "Record Attendance",
-    });
-    await expect(recordAttendanceButton).toBeVisible();
-
-    await recordAttendanceButton.click();
```

Comment:
If we want to click on something it’s usually not necessary to expect visibility before. The `.click()` call will wait for the element to show up